### PR TITLE
feat: reown on nano contracts

### PR DESCRIPTION
**Pull request by andreabadesso**
### Acceptance Criteria
- We should disable reown when the network does not support nano contracts
- We should add a missing key in allow-scripts

### Security Checklist
- [X] Make sure you do not include new dependencies in the project unless strictly necessary and do not include dev-dependencies as production ones. More dependencies increase the possibility of one of them being hijacked and affecting us.

### CODE REVIEW

File: src/sagas/reown.js [hunk 1]
```diff
@@ -130,12 +130,15 @@ function* isReownEnabled() {
 }
 
 function* init() {
-  const walletStartState = yield select((state) => state.walletStartState);
-
-  if (walletStartState !== WALLET_STATUS.READY) {
```
Comment (tuliomir) [inline src/sagas/reown.js:135]: chore: We can remove `WALLET_STATUS` from the imports (line 75) since we removed the only places where it was being read.

Comment (pedroferreira1) [inline src/sagas/reown.js:135]: Linter warned that `WALLET_STATUS` is not used anymore

File: src/reducers/reducer.js [hunk 2]
```diff
@@ -747,10 +747,7 @@ const onSetUnleashClient = (state, { payload }) => ({
 
 const onSetServerInfo = (state, action) => ({
   ...state,
-  serverInfo: {
```
Comment (pedroferreira1) [inline src/reducers/reducer.js:750]: We should update the default redux value for `serverInfo` and the docstring

File: src/actions.js [hunk 3]
```diff
@@ -310,8 +310,8 @@ export const setIsOnline = (status) => ({ type: types.SET_IS_ONLINE, payload: st
  * version {str} version of the connected server (e.g., 0.26.0-beta)
  * network {str} network of the connected server (e.g., mainnet, testnet)
  * */
```
Comment (pedroferreira1) [inline src/actions.js:312]: We should update the args docstring

File: src/sagas/wallet.js [hunk 4]
```diff
@@ -254,10 +254,14 @@ export function* startWallet(action) {
   yield put(setUniqueDeviceId(uniqueDeviceId));
 
   try {
-    yield call(wallet.start.bind(wallet), {
+    // XXX: This comes as undefined when the facede is the wallet-service.
```
Comment (tuliomir) [inline src/sagas/wallet.js:257]:
```suggestion
    // XXX: This comes as undefined when the facade is the wallet-service.
```

File: src/sagas/reown.js [hunk 5]
```diff
@@ -178,11 +181,15 @@ function* init() {
     // Pass extend = true so session expiration date get renewed
     yield call(refreshActiveSessions, true);
     yield fork(listenForAppStateChange);
-    yield fork(listenForNetworkChange);
+    // yield fork(listenForNetworkChange);
```
Comment (pedroferreira1) [inline src/sagas/reown.js:184]: Remove

**Review by pedroferreira1 — APPROVED** [review]

**Review by tuliomir — APPROVED** [review]

File: src/reducers/reducer.js (hunk 2 above)
Comment (andreabadesso) [reply src/reducers/reducer.js:750]: Done, thanks

File: src/sagas/reown.js (hunk 5 above)
Comment (andreabadesso) [reply src/sagas/reown.js]: Fixed, thanks

File: src/sagas/reown.js (hunk 1 above)
Comment (andreabadesso) [reply src/sagas/reown.js:135]: Fixed, thanks

File: src/sagas/wallet.js (hunk 4 above)
Comment (andreabadesso) [reply src/sagas/wallet.js]: Fixed, thanks

**Pull request merged**